corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -339,8 +378,221 @@ def _get_table_size(config):
             for datasource in static_datasources + dynamic_datasources
         ], key=lambda r: r[-1] if r[-1] != 'Table not found' else 0)
 
+        resource_model['datasources_info'] = rows
+
+    def collect_case_transactions(self):
+        total_form_case_updates = 0
+        total_forms = 0
+        total_transactions = 0
+
+        for db_name in get_db_aliases_for_partitioned_query():
+            db_cursor = connections[db_name].cursor()
+
+            with db_cursor as cursor:
+                cursor.execute("""
+                    SELECT COUNT(*) as num_forms, sum(d.count) as num_updates
+                    FROM (
+                        SELECT COUNT(*) as count
+                        FROM form_processor_casetransaction t
+                        JOIN form_processor_commcarecasesql c on t.case_id = c.case_id
+                        WHERE c.domain = %s
+                        GROUP BY form_id
+                    ) AS d
+                """, [self.domain])
+                result = cursor.fetchall()
+                forms, form_case_updates = (0, 0)
+                if result:
+                    forms, form_case_updates = result[0]
+
+                total_forms += forms
+                total_form_case_updates += form_case_updates
+
+                cursor.execute("""
+                    SELECT COUNT(*) as count
+                    FROM form_processor_casetransaction t
+                    JOIN form_processor_commcarecasesql c on t.case_id = c.case_id
+                    WHERE c.domain = %s
+                """, [self.domain])
+                (transactions,) = cursor.fetchone()
+                total_transactions += transactions
```

Comment:
how is this different from `total_form_case_updates`?

Reply:
@snopoke Mmm...it isn't.